client-side/src/stories/date/GenericDate.jsx

```diff
@@ -0,0 +1,49 @@
+
+import React, { useState, useEffect } from 'react';
+import { format } from 'date-fns';
+import Select from '../Select/Select';
+import GenericButton from '../Button/GenericButton.jsx'
+import axios from 'axios';
+import {formatDate} from './formatDate.js'
+
+const DateFormatter = ({ initialDate }) => {
+  const [currentDate, setCurrentDate] = useState(format(initialDate, 'yyyy-MM-dd'));
```

Comment:
please take the default formatDate from the enum of the formats - define of them as default.